datastream-server/src/main/java/com/linkedin/datastream/server/zk/ZkAdapter.java

```diff
@@ -1434,6 +1434,13 @@ public static String parseHostnameFromZkInstance(String instance) {
     return instance.substring(0, instance.lastIndexOf('-'));
   }
 
+  /**
+   * Parse the Zk instance (ex. hostname-0000) into liveinstance
```

Comment:
nit: reword
Parse the liveinstance ID from the owner string (format: hostname-<liveinstance id>, ex. hostname-0000)